### PR TITLE
DEVELOPER-1243 Fixed Buzz.js off by one on the dcp query for results.

### PR DESCRIPTION
**Pull request by Dantheman720**
Kind of a hacked fix, but it fixes the off by one issue

### CODE REVIEW

File: javascripts/buzz.js [hunk 1]
```diff
@@ -224,6 +224,10 @@ app.buzz = {
 
           app.buzz.infiniteScrollCalled = true;
           var from = $('.buzz-item').length + 1;
+        
+          if($('.buzz-item').length == 4){
```
Comment (paulrobinson) [inline javascripts/buzz.js]: I think you have just moved the gap out to the next request. I suspect you are now getting items:

    0-3
    4-11
    13-20

Where has item 12 gone?

File: javascripts/buzz.js [hunk 2]
```diff
@@ -224,6 +224,10 @@ app.buzz = {
 
           app.buzz.infiniteScrollCalled = true;
           var from = $('.buzz-item').length + 1;
```
Comment (paulrobinson) [inline javascripts/buzz.js]: I think the real fix is to change this to:

    var from = $('.buzz-item').length

Comment (Dantheman720) [inline javascripts/buzz.js]: Yeah, it just delays the problem by adding the if length == 4

**Comment by Dantheman720** [comment]
restest this please

**Pull request merged**